src/components/app/header.tsx

```diff
@@ -36,7 +36,7 @@ const Header: FunctionComponent = () => {
         {viewer ? (
           <div className={className}>
             {children}
-            <OnlinePresenceCTA variant="header" />
+            {viewer.is_pro && <ProjectClubCTA variant="header" />}
```

Comment:
I want to use the header for this CTA, but want to make sure it's only members.

@MaggieAppleton I think this CTA is an interesting spot for some light personalization. We could have logic here that decided what we wanted to ask the user to do right now.

Lots of potential.

Reply:
Nice. Making it only members makes sense.

Reply:
love how easy it is to check for viewer status